client/blocks/reader-full-post/index.jsx

```diff
@@ -353,14 +355,38 @@ export class FullPostView extends React.Component {
 	}
 }
 
+const ConnectedFullPostView = connect(
+	( state, ownProps ) => {
+		const {
+			site_ID: siteId,
+			feed_ID: feedId,
+			isExternal: is_external
+		} = ownProps.post;
+
+		const props = {};
+
+		if ( ! is_external && siteId ) {
+			props.site = getSite( state, siteId )
+		}
+		if ( feedId ) {
+			props.feed = getFeed( state, feedId );
+		}
+		return props;
+	},
+	dispatch => {
+		return bindActionCreators( {
+			setSection
+		}, dispatch );
+	}
```

Comment:
The second argument can be [shortened](https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments) to ```{ setSection }```.